workshop/main.lua

```diff
@@ -60,8 +60,8 @@ return function(dev)
 
     -- If initialize phase fails, prompt to the error screen
     if (not success) then
-        teverse.interface:destroyChildren()
-        
+        teverse.interface:destroyAllChildren()
```

Comment:
did you mean destroyChildren

Reply:
Correct. It was a previous merge conflict error that got overwritten by the branch being out of date. It has been corrected in https://github.com/teverse/teverse/pull/73/commits/46ae4c4ba7eb62e28dc460699dae3f17c23d2725 .